projects/vdk-plugins/vdk-ipython/setup.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import setuptools
+
+
+__version__ = "0.1.0"
+
+setuptools.setup(
+    name="vdk-ipython",
+    version=__version__,
+    url="https://github.com/vmware/versatile-data-kit",
+    description="Ipython extension for VDK",
+    long_description=pathlib.Path("README.md").read_text(),
+    long_description_content_type="text/markdown",
+    install_requires=["vdk-core"],
```

Comment:
Seems suspicious that ipython is missing as dependency here. 

It's explained in the requirements.txt comment : 
```
this [requirements.txt] file is used to provide testing requirements
for requirements (dependencies) needed during and after installation of the plugin see (and update) setup.py install_requires section
```